model.go

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2023 Cisco and/or its affiliates.
```

Comment:
I think we need to rework this with using openapi as we planned.

Reply:
Created a task for this: https://github.com/networkservicemesh/cmd-dashboard-backend/issues/2